flocker/control/schema/types.yml

```diff
@@ -280,3 +280,36 @@ definitions:
       primary:
         '$ref': '#/definitions/primary'
     additionalProperties: false
+
+  lease_expiration:
+    title: "Lease Expiration"
+    description: |
+      The number of seconds until a lease expires, or null to indicate no expiration.
+    type:
+      - "number"
+      - "null"
+
+  lease_dataset_id:
```

Comment:
Which suggests a bigger problem but not one I'm going to think too hard about for now.